karabo/examples/SRCNet_workflow_AAstar.py

```
@@ -0,0 +1,284 @@
+# This script generates simulated visibilities and images resembling SKAO data.
+# It also outputs corresponding ObsCore metadata ready to be ingested to Rucio.
+#
+# Images: dirty image and cleaned image using WSClean.
+# These are MFS images (frequency channels aggregated into one channel),
+# not full image cubes.
+#
+# Size of generated data with default settings should be around 3 TB:
+# - 1.5 TB visibilities (before image cleaning)
+# - 3 TB visibilities (after image cleaning)
+# - 12 GB images
+import math
+import os
+from datetime import datetime, timedelta
+
+import numpy as np
+import pandas as pd
+from astropy import constants
+
+from karabo.data.obscore import ObsCoreMeta
+from karabo.data.src import RucioMeta
+from karabo.imaging.image import Image
+from karabo.imaging.imager_base import DirtyImagerConfig
+from karabo.imaging.imager_wsclean import (
+    WscleanDirtyImager,
+    WscleanImageCleaner,
+    WscleanImageCleanerConfig,
+)
+from karabo.simulation.interferometer import InterferometerSimulation
+from karabo.simulation.observation import Observation
+from karabo.simulation.sky_model import SkyModel
+from karabo.simulation.telescope import Telescope
+from karabo.simulation.telescope_versions import SKAMidAAStarVersions
+from karabo.simulation.visibility import Visibility
+from karabo.simulator_backend import SimulatorBackend
+from karabo.util.gpu_util import is_cuda_available
+from karabo.util.helpers import Environment
+
+# Simulation
+# Phase center: Must be inside coverage region of selected sky
+SKY_MODEL = Environment.get("SKY_MODEL", str, "MIGHTEE_L1")
+PHASE_CENTER_RA_DEG = Environment.get("PHASE_CENTER_RA_DEG", float, 150.12)
+PHASE_CENTER_DEC_DEG = Environment.get("PHASE_CENTER_DEC_DEG", float, 2.21)
+START_FREQ_HZ = Environment.get("START_FREQ_HZ", float, 1.304e9)
```

Possible issue here: Is there a check down the line that the phase center coordinates are within the sky model? If they're not the simulation will run for hours but will not generate meaningful output. It could be user friendly to 'fail fast' here.